model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -208,7 +216,7 @@ def _determine_trials_based_on_max_percentage_of_search_space(self) -> int:
 
         return max_trials_based_on_percentage_of_search_space
 
-    def _decide_between_percentage_and_trial_count(
+    def _decide_max_between_percentage_and_trial_count(
```

Comment:
I think function names should be based on what they accomplish (determine_max_configs_to_search) instead of HOW they do it (deciding between two values)